provider/zencoder/zencoder.go

```diff
@@ -89,11 +91,41 @@ func (z *zencoderProvider) buildOutputs(job *db.Job, transcodeProfile provider.T
 		if err != nil {
 			return nil, fmt.Errorf("Error building output: %s", err.Error())
 		}
+		if zencoderOutput.Format == "ts" {
+			hlsOutputs++
+		}
 		zencoderOutputs = append(zencoderOutputs, &zencoderOutput)
 	}
+	if hlsOutputs > 0 {
+		outputsWithHLS := make([]*zencoder.OutputSettings, len(zencoderOutputs)+1)
+		copy(outputsWithHLS, zencoderOutputs)
+		hlsPlaylist := z.buildHLSPlaylist(zencoderOutputs, hlsOutputs)
+		outputsWithHLS[len(zencoderOutputs)] = &hlsPlaylist
+		zencoderOutputs = outputsWithHLS
+	}
 	return zencoderOutputs, nil
 }
 
+func (z *zencoderProvider) buildHLSPlaylist(outputs []*zencoder.OutputSettings, hlsOutputs int) zencoder.OutputSettings {
+	output := zencoder.OutputSettings{
+		BaseUrl:  outputs[0].BaseUrl,
+		Filename: "master.m3u8",
```

Comment:
The master playlist file name is currently defined by ``TranscodeProfile.StreamingParams.PlaylistFileName``. Not sure if this is the best approach, but we should make it the same across all providers :D